Moda.Web/src/moda.web.reactclient/src/app/planning/roadmaps/components/roadmaps-timeline.tsx

```diff
@@ -30,15 +28,12 @@ export interface RoadmapsTimelineProps {
   viewSelector?: React.ReactNode | undefined
 }
 
-interface RoadmapTimelineItem extends ModaDataItem<RoadmapChildrenDto> {
-  id: number
-  end: Date
-  type: string
-  //roadmap?: RoadmapChildrenDto
+type RoadmapTimelineObject = RoadmapChildrenDto & {
```

Comment:
I old interface required that id, end, and type were required, but the type doesn't.  Also, I don't think the roadmap object is needed for the type.

Reply:
I'm also confused why this is RoadmapChildrenDto & {} instead of ModaDataItem.

Reply:
Fixed